src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Models/ResourcePolicy.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Altinn.Platform.Authorization.Models
+{
+    /// <summary>
+    /// This model describes the actions that can be performed for a particular resource within a app policy
+    /// </summary>
+    public class ResourcePolicy
+    {
+        /// <summary>
+        /// Gets or sets a description of the resource policy.
+        /// </summary>
+        public LocalizedText Description { get; set; }
```

Comment:
Have you considered the case where other languages are supported? Could the resourcePolicy have an id so that we can lookup the description and title at a later point in time?

Reply:
Resource in this context can be a task or event as well; we have not defined a scalar way to to uniquely identify a particular task or event within an app. `org1/app1/subresource1` is ambiguous as we do not know the attribute id of `subresource1`. Therefore the API uses `List<AttributeMatch>` in order to uniquely identify resources. cc @jonkjetiloye

Reply:
How will the user flow look here? Will the retrieved policies be cached in AltinnII for a longer period of time? For messagebox elements language is included in the request so that the resource has the texts populated with the user's language before returning. By hard coding NN, NB and EN in the model we make it harder to support other languages in the future. Consider a regular dictionary and using nn, nb and en as keys.. This way we don't require updates to the model if other languages are supported in the future.

Reply:
This will most likely be cached In Altinn2 to reduce network roundtrips. Agree that using a dictionary makes more sense, perhaps with a default language field akin to what we already have for apps/delegationSchemes In Altinn2 authorization.

Reply:
I replaced LocalizedText with string and added a queryparameter "language" to the controller, then removed the LocalizedText model